scene/main/viewport.cpp

```diff
@@ -1429,7 +1429,7 @@ String Viewport::_gui_get_tooltip(Control *p_control, const Vector2 &p_pos, Cont
 	String tooltip;
 
 	while (p_control) {
-		tooltip = p_control->get_tooltip(pos);
+		tooltip = p_control->atr(p_control->get_tooltip(pos));
```

Comment:
> This change is missing if you were to supersede my PR:
https://github.com/godotengine/godot/pull/81180/files#diff-f7fb6c2c4c5d4fce35e389297048f54be3d4b08621edfc55a5586b437031ff10R51-R59
You already added TRANSLATION_CHANGED to ColorPicker, so you could handle this as well.

Not necessary thanks to this change.